### DISCUSSION TITLE
Convite para fazer parte da Angola Open-source Community Mentorship

### DISCUSSION THREAD

**Discussion by antonio-pedro99** [moclojer/moclojer#137] [Announcements]
Olá, equipe [moclojer](https://github.com/moclojer)

Espero que estejam bem. Sou Antonio Pedro da [Angola Open-source Community](https://github.com/angolaosc), uma comunidade de desenvolvedores dedicados a promover o desenvolvimento open-source em Angola.

Estamos atualmente organizando um Programa de Mentoria em Desenvolvimento Open-source em Angola, visando capacitar os nossos desenvolvedores locais e estimular a colaboração com projetos de código aberto de renome internacional.

Gostaríamos de convidar o [moclojer](https://github.com/moclojer/moclojer) para participar como uma organização mentora no nosso programa. Acreditamos que a participação do [moclojer](https://github.com/moclojer/moclojer) como mentor pode ser extremamente valiosa para nossos desenvolvedores, permitindo que eles contribuam ativamente para um projeto relevante e aprendam com as melhores práticas de uma organização experiente.

O nosso [programa](https://github.com/angolaosc/mentorship) terá duração de um mês e contará com diferentes etapas, incluindo pré-mentoria(onde nós a organização, iremos conduzir um workshop introdutório sobre open-source), mentoria, onde os desenvolvedores terão a oportunidade de trabalhar em qualquer issue aberta neste projeto, receber feedback de especialistas e desenvolver as suas habilidades em desenvolvimento de software.

Seria uma honra contar com o [moclojer](https://github.com/moclojer/moclojer) como um mentor no nosso programa. Acreditamos que essa colaboração pode trazer benefícios mútuos, fortalecendo a comunidade open-source em Angola e promovendo a inovação tecnológica em ambos os lados.

Se estiverem interessados em participar ou tiverem-se alguma dúvida, por favor, deixem-nos saber. Estamos ansiosos para uma possível parceria com o [moclojer](https://github.com/moclojer/moclojer).

Agradecemos a sua atenção e consideração.

Atenciosamente,
Antonio Pedro
Angola Open-source Community

**Comment by matheusfrancisco** [comment]
Hey @antonio-pedro99 como vai? Que iniciativa legal. Vamos conversar tenho interesse em contribuir com a comunidade sim. 
Poderia  enviar um email para matheus@cljazz.co

Caso tenham interesse o moclojer é desenvolvido pelo pessoal da cljazz 

https://github.com/cljazz  
https://cljazz.co

Também estamos começando um novo projeto open source https://github.com/cljazz/supabase-clj caso tenham interesse em contribuir. 

Att
Matheus Francisco

**Reply by antonio-pedro99** [reply]
Ola @matheusfrancisco, obrigado pelo retorno!
Deixarei um email agora mesmo com mais detalhes sobre o programa.

Obrigado pela recomendação, abrirei uma discussão no novo projeto com o convite. 

Atenciosamente, 
Antonio Pedro.

**Comment by antonio-pedro99** [comment]
@matheusfrancisco deixei um email.

**Reply by matheusfrancisco** [reply]
Acabei de responder seu email. Agradeço o interesse em ajudar o moclojer.
Porém devido algumas viagens  minhas esse mês  não  conseguirei participar, porém me deixo a disposição  para logo após  parearmos em algumas issues aqui do moclojer, caso vocês  tenham interesse.

Outro ponto, tentei acessar o discord da comunidade e o link não  estava válido.
